charts/eks-anywhere-packages/templates/credentialpackage.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   namespace: eksa-packages-{{.Values.clusterName}}
   annotations:
     "helm.sh/resource-policy": keep
+    "skip-webhook-validation": "true"
```

Comment:
Can you rename this to something like `anywhere.eks.aws.com/internal`?

Reply:
done

Reply:
I didn't see an updated commit for this one